var/spack/repos/builtin/packages/py-onnxruntime/package.py

```diff
@@ -99,10 +129,28 @@ class PyOnnxruntime(CMakePackage, PythonExtension):
     root_cmakelists_dir = "cmake"
     build_directory = "."
 
+    def patch(self):
+        if self.spec.satisfies("@1.7: + rocm"):
```

Comment:
can you change it to self.spec.satisfies("@1.17 +rocm ")

Reply:
Done! Thanks for catching that